snap/snapcraft.yaml

```diff
@@ -193,28 +193,33 @@ parts:
       versions_url=https://raw.githubusercontent.com/kata-containers/runtime/${SNAPCRAFT_PROJECT_VERSION}/versions.yaml
       version="$(curl -sSL ${versions_url} | yq r - assets.kernel.version | tr -d v)"
       x_version="$(echo $version | sed -e 's|.[[:digit:]]*$||').x"
-      case "$(arch)" in
-        "x86_64")
-          config="x86_64_kata_kvm_${x_version}"
-        ;;
-
-        "ppc64le")
-          config="powerpc_kata_kvm_${x_version}"
-        ;;
-
-        "aarch64")
-          config="arm64_kata_kvm_${x_version}"
-        ;;
-
-        "s390x")
-          config="s390_kata_kvm_${x_version}"
-        ;;
-
-        *)
-          echo "ERROR: Unsupported architecture $(arch)"
-          exit 1
-        ;;
-      esac
+      archfragdir="${SNAPCRAFT_STAGE}/kernel/configs/fragments/$(arch)"
+      if [ -f "${archfragdir}/.config" ]; then
+        config="fragments/$(arch)/.config"
```

Comment:
who is going to generate this file?

Reply:
The build script will generate this file on the fly and copy it into the location referenced above. This should be re-generated every time we run the test.

Reply:
@ganeshmaharaj - is this related to https://github.com/kata-containers/tests/pull/1490 as well?
If so, is there some dependence/sequence we need to land the relevant PRs in?

Reply:
Should not affect the stable branches for now @grahamwhaley. This should be alright to go standalone before kernel fragment patch.